plugin/completion.py

```diff
@@ -105,8 +104,8 @@ def handle_resolve_response(self, response, view):
                 sel.clear()
                 sel.add(current_completion.region)
                 view.run_command("insert_snippet", {"contents": insertText})
-            except Exception as err:
-                exception_log("Error inserting snippet: " + insertText, err)
+            except Exception:
+                log.exception("Error inserting snippet: %s", insertText)
 
```

Comment:
This will have the same effect as the older function call `exception_log("Error inserting snippet: " + insertText, err)`. The `log.exception()` can only be used inside exceptions as it catches the exception by himself to print the stack trace.